URLs.md

```diff
@@ -116,6 +116,16 @@ git checkout -b admin internal/admin
 For browsing `internal/admin` branch on web:
 https://cgit.freebsd.org/${repo}/log/?h=internal/admin
 
+For pushing, either specify the full refspec:
+```
+git push origin HEAD:refs/internal/admin
+```
+
+Or setting `push.default` gitconfig to use the `upstream`:
+```
+git config push.default upstream
```

Comment:
mixed feelings about this. likely safe enough. will go along with group consensus.

Reply:
I've tested in my end, pushed to the `main` and `internal/admin` branches, may need to test more with src workflow like vendor import and MFC.